tests/test_upload_manager.py

```diff
@@ -32,5 +37,68 @@ def test_upload_manager_switches_drivers():
     container.bind('StorageConfig', storage)
     container.bind('UploadDiskDriver', object)
     container.bind('UploadTestDriver', object)
-
     container.bind('UploadManager', UploadManager(container).driver('test'))
+
+
+class ImageTest():
+    """
+    Image test for emulate upload file
+    """
+
+    @property
+    def filename(self):
+        return os.getcwd() + "/tests/static/test.jpg"
+
+    @property
+    def file(self):
+        return open(self.filename, 'rb')
+
+
+def test_upload_file():
+    """
+    This test is responsible for checking if you upload a file correctly.
+    """
+
+    container = App()
+
+    container.bind('Application', application)
+    container.bind('StorageConfig', storage)
+    container.bind('UploadDiskDriver', UploadDiskDriver)
+
+    img = ImageTest()
+    UploadManager(container).driver('disk').store(img)
```

Comment:
All tests need an assertion so just stick `assert` in the front like so: `assert UploadManager(container).driver('disk').store(img)`